backend-ts/Dockerfile

```diff
@@ -1,7 +1,10 @@
 FROM node:16.13 as backend-ts
 # Installing packages...
 WORKDIR /backend-ts
-COPY . .
+COPY package.json .
+COPY package-lock.json .
 RUN npm install
+# Copy source files after installing packages
+COPY . .
```

Comment:
The re-ordering in here prevents rebuilding the packages layer when only source code changes are made. Same change as we did on #51.